### ISSUE TITLE
Flameshot not starting.

### ISSUE THREAD

**Issue by Retold3202** [netblue30/firejail#6094]
### Description

Flameshot doesn't start under Fedora 39.

### Steps to Reproduce

1. Run in bash LC_ALL=C firejail flameshot (LC_ALL=C to get a consistent
    output in English that can be understood by everybody)
2. The application doesn't start.

### Expected behavior

Application starts.

### Actual behavior

Application does not start.

### Behavior without a profile

_What changed calling `LC_ALL=C firejail --noprofile /path/to/program` in a
terminal?_

The application starts and functions without any issue.

### Environment

- Fedora 39
- Firejail version 0.9.73
- GNOME 45.1

### Checklist

- [x] The issues is caused by firejail (i.e. running the program by path (e.g. `/usr/bin/vlc`) "fixes" it).
- [x] I can reproduce the issue without custom modifications (e.g. globals.local).
- [x] The program has a profile. (If not, request one in `https://github.com/netblue30/firejail/issues/1139`)
- [x] The profile (and redirect profile if exists) hasn't already been fixed [upstream](https://github.com/netblue30/firejail/tree/master/etc).
- [x] I have performed a short search for similar issues (to avoid opening a duplicate).
  - [x] I'm aware of `browser-allow-drm yes`/`browser-disable-u2f no` in `firejail.config` to allow DRM/U2F in browsers.
- [ ] I used `--profile=PROFILENAME` to set the right profile. (Only relevant for AppImages)

### Log

<details>
<summary>Output of <code>LC_ALL=C firejail /path/to/program</code></summary>
<p>

```
(flameshot:26): dbind-WARNING **: 16:06:08.518: AT-SPI: Error retrieving accessibility bus address: org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
dbus[26]: D-Bus library appears to be incorrectly set up: see the manual page for dbus-uuidgen to correct this issue. (Failed to open "/var/lib/dbus/machine-id": No such file or directory; Failed to open "/etc/machine-id": No such file or directory)
  D-Bus not built with -rdynamic so unable to print a backtrace

```

</p>
</details>

<details>
<summary>Output of <code>LC_ALL=C firejail --debug /path/to/program</code></summary>
<p>

```
https://gist.github.com/Retold3202/2344531594cf63718b707595bcfbe6a5

```

</p>
</details>

**Comment by rusty-snake** [comment]
I know. 5d0822c52c9a5e631676899e9642911d9143dba8 remove `/etc/machine-id` for a lot program that actually need it.

**Comment by Retold3202** [comment]
> I know. [5d0822c](https://github.com/netblue30/firejail/commit/5d0822c52c9a5e631676899e9642911d9143dba8) remove `/etc/machine-id` for a lot program that actually need it.

Yep, I can confirm changing `private-etc @tls-ca` to `private-etc @tls-ca,machine-id` in the profile appears to have fixed the problem.
